views/artist-search-results.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="search">
+    {{#each artist}}
+    <div class="card">
+        <img class="card-img-top" src={{images.[0].url}} alt="artist album">
```

Comment:
When array items are objects, you can just access that object using the dot notation in Handlebars. Ex: images.0.url

Reply:
Thanks for the note Filipe 😃will put it in practice from now on